### DISCUSSION TITLE
How to install a .application file?

### DISCUSSION THREAD

**Discussion by timothyli96** [The-Wineskin-Project/WineskinServer#286] [Q&A]
I'm trying to install a piece of windows software but it comes in a .application format instead of an exe.

Wondering if there is a way to install this.

Thanks!

![image](https://user-images.githubusercontent.com/67601147/181281928-fc312c12-2b65-4166-b78d-098890414454.png)

**Comment by easp** [comment]
https://wiki.winehq.org/FAQ#How_do_I_install.2Frun_a_ClickOnce_.28.application.29_file.3F
